core/src/main/kotlin/gropius/service/user/AvatarGenerationService.kt

```diff
@@ -27,7 +29,7 @@ class AvatarGenerationService {
      * @return The avatar as Base64 encoded SVG data url
      */
     fun generateAvatar(id: String): String {
-        val numFromName = id.hashCode()
+        val numFromName = max(abs(id.hashCode()), 0)
```

Comment:
why not https://docs.oracle.com/javase/8/docs/api/java/lang/Integer.html#toUnsignedLong-int- to preserve highest bit?